src/get-default-blueprint-name-override.js

```diff
@@ -1,22 +1,32 @@
 'use strict';
 
+const fs = require('fs');
+const path = require('path');
 const { spawn } = require('./run');
 
 /**
  * Run npm view to get the package.json and parse it to check if it contains property
  * to override the default blueprint name
  *
- * @param {string} packageName - Name of package
+ * @param {string} packageNameOrPath - Name of package i.e. ember-cli-package-name or path to a local package
+ * @param {string} cwd - Default to the process's CWD. Parent functions can override
  * @returns {Promise<Array<string>>}
  */
-module.exports = async function getBlueprintNameOverride(packageName) {
-  let ps = await module.exports.spawn('npm', ['view', packageName, '--json']);
+module.exports = async function getBlueprintNameOverride(packageNameOrPath, cwd = process.cwd()) {
+  let localPackageJsonPath = path.join(path.resolve(cwd, packageNameOrPath), 'package.json');
+  let packageJsonStr;
 
-  if (ps.exitCode > 0) {
-    return null;
+  if (fs.existsSync(localPackageJsonPath)) {
+    packageJsonStr = fs.readFileSync(localPackageJsonPath);
+  } else {
+    let ps = await module.exports.spawn('npm', ['view', packageNameOrPath, '--json']);
```

Comment:
I think we are still missing coverage of this line. Can you publish something like
https://github.com/kellyselden/ember-cli-update-npm-addon-test
https://www.npmjs.com/package/ember-cli-update-npm-addon-test
and then we will be able to actually test it. Something like `ember-cli-update-default-blueprint-override-test` should be good.

Reply:
Created `ember-cli-update-default-blueprint-override-test`: https://www.npmjs.com/package/ember-cli-update-default-blueprint-override-test

Added test case to read that. I typically try to avoid tests with network dependencies so it took me a while to understand what you meant.

Reply:
Thanks! Do you mind doing `npm owner add` for me?